docs/topics/api/shelves.rst

```diff
@@ -117,3 +117,18 @@ The current implemenation forwards these impression pings to Adzerk.
 .. http:post:: /api/v4/shelves/sponsored/impression/
 
     :form string impression_data: the signed data payload that was sent in the :ref:`sponsored shelf <sponsored-shelf>` response.
+
+
+---------------------------
+Sponsored Shelf Clicks
+---------------------------
+
+.. _sponsored-shelf-click:
+
+When and item on then sponsored shelf is clicked on by the user to navigate to the detail page this endpoint should be used to record the click.
```

Comment:
s/then sponsored/the sponsored

Reply:
I just noticed there was a second error too, which you might be referencing. The inital "and" should be an "an".

Reply:
https://github.com/mozilla/addons-server/pull/15704/commits/8ef199ba322cc5554edeb055e4f92fee7e068854